ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/management-gnmi.md

```diff
@@ -59,10 +59,10 @@ interface Management1
 ```eos
 !
 management api gnmi
-   transport grpc MGMT
+   transport grpc {'access_group': 'ACL-GNMI', 'name': 'MGMT'}
```

Comment:
@Shivani-chourasiya, this appears like an issue in the template. The output shouldn't change here.